pkgs/applications/audio/bitwig-studio/bitwig-studio5.nix

```diff
@@ -27,11 +27,12 @@
 
 stdenv.mkDerivation rec {
   pname = "bitwig-studio";
-  version = "5.1.8";
+  version = "5.1.9";
 
   src = fetchurl {
-    url = "https://downloads.bitwig.com/stable/${version}/${pname}-${version}.deb";
-    sha256 = "sha256-KxNLae/uTYL1m/X+/7wr7hhKfw31NpB9Mw9RzfrTuus=";
+    url = "https://www.bitwig.com/dl/Bitwig%20Studio/${version}/installer_linux/";
+    curlOpts = "--location";
```

Comment:
You don't need to add `--location` to `curlOpts`. `fetchurl` uses it by default anyway: https://github.com/NixOS/nixpkgs/blob/e0ba5d6aaf27ed0deb1ad168bd151a6fd23634f1/pkgs/build-support/fetchurl/builder.sh#L12-L14

Reply:
Removed - thanks!